command/daemon/config.go

```diff
@@ -66,10 +66,12 @@ type Config struct {
 	}
 
 	Resources struct {
-		LimitCPU      int64     `envconfig:"DRONE_RESOURCE_LIMIT_CPU"`
-		LimitMemory   BytesSize `envconfig:"DRONE_RESOURCE_LIMIT_MEMORY"`
-		RequestCPU    int64     `envconfig:"DRONE_RESOURCE_REQUEST_CPU" default:"100"`
-		RequestMemory BytesSize `envconfig:"DRONE_RESOURCE_REQUEST_MEMORY" default:"104857600"` // default 100MB
+		LimitCPU       int64     `envconfig:"DRONE_RESOURCE_LIMIT_CPU"`
+		LimitMemory    BytesSize `envconfig:"DRONE_RESOURCE_LIMIT_MEMORY"`
+		LimitStorage   BytesSize `envconfig:"DRONE_RESOURCE_LIMIT_STORAGE"`
+		RequestCPU     int64     `envconfig:"DRONE_RESOURCE_REQUEST_CPU" default:"100"`
+		RequestMemory  BytesSize `envconfig:"DRONE_RESOURCE_REQUEST_MEMORY" default:"104857600"` // default 100MB
+		RequestStorage BytesSize `envconfig:"DRONE_RESOURCE_REQUEST_STORAGE"`
 		// Defaults for min memory & cpu requests are set to ensure that default limitrange values are not used.
```

Comment:
I didn't add a min ephemeral storage request, but can if you like? I'm not sure I follow why this overrides the default LimitRange (if present). That's what we currently use to set the `ephemeral-storage` for the whole namespace in lieu of doing it per-pipeline.

Reply:
Also `DRONE_RESOURCE_REQUEST_EPHEMERAL_STORAGE` got shortened to `DRONE_RESOURCE_REQUEST_STORAGE`, happy to change.